### PR TITLE
ServiceLB Reword

### PR DESCRIPTION
**Pull request by dereknola**
Supersede https://github.com/k3s-io/docs/pull/75 with DCO signed (includes credit to original author)

### CODE REVIEW

File: docs/networking/networking.md [hunk 1]
```diff
@@ -57,13 +57,13 @@ Create a [Service of type LoadBalancer](https://kubernetes.io/docs/concepts/serv
 
 ### Controlling ServiceLB Node Selection
 
-To exclude nodes from being used by ServiceLB, add the following label to the nodes that should host ServiceLB Pods. All unlabeled nodes will not be used for ServiceLB.
+In order for ServiceLB to assign nodes to pools, nodes must have the following label:
```
Comment (brandond) [inline docs/networking/networking.md:60]: We still need the prior language explaining that adding the `enablelb` annotation to any node disables it on any unlabeled nodes. We just need to add a bit about `lbpool` also requiring `enablelb` on any pooled nodes.

**Review by brandond — CHANGES_REQUESTED** [review]

File: docs/networking/networking.md (hunk 1 above)
Comment (brandond) [inline docs/networking/networking.md:60]:
```suggestion
To exclude nodes from being used by ServiceLB, add the following label to the nodes that should host ServiceLB Pods. Nodes that remain unlabeled will not be used for ServiceLB.
```

File: docs/networking/networking.md [hunk 2]
```diff
@@ -57,13 +57,13 @@ Create a [Service of type LoadBalancer](https://kubernetes.io/docs/concepts/serv
...
 
 ```
 svccontroller.k3s.cattle.io/enablelb
 ```
 
-To select a particular subset of nodes to host pods for a LoadBalancer, set matching annotation values on the Nodes and Service. For example:
+To select a particular subset of nodes to host pods for a LoadBalancer, set matching label values on the Nodes and Service. For example:
```
Comment (brandond) [inline docs/networking/networking.md:66]:
```suggestion
To select a particular subset of nodes to host pods for a LoadBalancer, add the `svccontroller.k3s.cattle.io/enablelb` label to the desired nodes, and set matching label values on the Nodes and Services. For example:
```

File: docs/networking/networking.md [hunk 3]
```diff
@@ -57,13 +57,13 @@ Create a [Service of type LoadBalancer](https://kubernetes.io/docs/concepts/serv
...
 
 1. Label Node A and Node B with `svccontroller.k3s.cattle.io/lbpool=pool1`
 2. Label Node C and Node D with `svccontroller.k3s.cattle.io/lbpool=pool2`
```
Comment (brandond) [inline docs/networking/networking.md:69]:
```suggestion
1. Label Node A and Node B with `svccontroller.k3s.cattle.io/lbpool=pool1` and `svccontroller.k3s.cattle.io/enablelb=true`
2. Label Node C and Node D with `svccontroller.k3s.cattle.io/lbpool=pool2` and `svccontroller.k3s.cattle.io/enablelb=true`
```

**Review by brandond — CHANGES_REQUESTED** [review]

**Comment by dereknola** [comment]
I reworded the first part. I didn't like that we start with "this is how to turn this off", so I used and admonition to highlight that separately.
image

**Review by brandond — APPROVED** [review]

**Pull request merged**